### DISCUSSION TITLE
Model overfits greatly when given partial yearly data

### DISCUSSION THREAD

**Discussion by bwassie** [ourownstory/neural_prophet#784] [Q&A - get help using NeuralProphet]
Hi folks. I'm using NeuralProphet for time series forecasting at the daily level. I have around 3 years of data right now and if I feed the model 2.5 years of data as training, it overfits severely on the remaining set. However, if I fit it with only 2 years of complete data, the model is very accurate on the holdout set. Has anyone seen this behavior before? I'm not sure if it's because I have limited data or what. I'm tuning for most parameters (all regularization terms, n_lags, epochs, learning rate...etc) except for number of layers (fixed at 0). I'm attaching the time series below. I'm very perplexed by this and would appreciate some guidance. 

![image](https://user-images.githubusercontent.com/22499035/191673973-29dfbd22-465c-44d9-9877-e5f66d914da7.png)

**Comment by ourownstory** [comment]
Hi @bwassie this is interesting. 
Can you please provide a minimal code snippet to reproduce this?

(on a first guess: be careful when using a large number of lags and forecasts. the number of parameters is proportional to their product.)
